sensor_msgs/Image image_raw
---
lasr_vision_msgs/Detection[] detected_objects
